octo/data/dataset.py

```diff
@@ -525,7 +541,9 @@ def make_interleaved_dataset(
             train=train,
             num_parallel_calls=threads,
             num_parallel_reads=reads,
-            dataset_statistics=dataset_statistics,
+            dataset_statistics=dataset_statistics
+            if dataset_statistics is not None
+            else data_stats,
```

Comment:
Could you use more informative variable names to distinguish between the global `dataset_statistics` and the per-dataset `data_stats`? Otherwise LGTM!